spec/personas/persona_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+describe "personas" do
+  context "2023" do
+    let(:tax_year) { 2023 }
+
+    context "az" do
+      let(:state_code) { :az }
+
+      context "johnny" do
+        let(:persona_name) { "johnny" }
+        let(:submission_id) { "1234562024165nly30yy" }
```

Comment:
Use the `ignore_list` you implemented to avoid comparing this, instead of overriding, you doof